.github/workflows/arduino.yml

```diff
@@ -0,0 +1,30 @@
+name:  ESP32 board on CI
+on: [push, pull_request]
+jobs:
+  build-sketch:
+    runs-on: ubuntu-latest
+    
+    steps:
+       - name: Checkout
+         uses: actions/checkout@v2
+       - name: Build on Arduino CLI
+         run: bash ci/build-arduino.sh
+       - name: Setup Arduino CLI
+         uses: arduino/setup-arduino-cli@v1.0.0
+       - name: upload
+         uses: actions/upload-artifact@master
+         with:
+          name: app-version-${{ env.VERSION }}
+          path: app-${{ env.VERSION }}.exename:ESP32 board on CI
+ 
+       - name: Install platform
+         run: |
+          arduino-cli core update-index
+          arduino-cli core install ${{ matrix.arduino-platform }}
+       - name: Compile Sketch
+         run: arduino-cli compile --fqbn ${{ matrix.fqbn }} ./blink
+  
+
```

Comment:
I suggest you have a look at the relevant sections of the [sample repository](https://github.com/DIT112-V20/kotlin-app-arduino-sketch-ci) that was provided during the normal duration of the course.

Reply:
@ayah-abuowda have you had a look into the sample repository I mentioned above?

Reply:
yes I do

Reply:
Why not start by that example then instead of getting confused by things you find online, which may not be so relevant and are more complicated?

Reply:
should I delete it and do it again?

Reply:
Does the current work satisfy the acceptance criteria of #32?

Particularly this one:

> Arduino sketch is built for an ESP32 board on CI

Reply:
yes its work

Reply:
I'll leave some comments on the latest version. Let's continue the discussion there.